charts/agent-control-deployment/templates/preinstall-job-register-system-identity.yaml

```diff
@@ -290,7 +290,7 @@ spec:
                 {{ include "newrelic-agent-control.auth.secret.name" . }} \
                 --from-literal=CLIENT_ID=$CLIENT_ID \
                 --from-file="private_key=$TEMPORAL_FOLDER/key" | \
-              jq '.metadata.labels += ({{ include "newrelic.common.labels" . | fromYaml | toJson }} + {"app.kubernetes.io/created-by": "ac-auth-job"})' | \
+              jq '.metadata.labels += ({{ include "newrelic.common.labels" . | fromYaml | toJson }} + {"app.kubernetes.io/managed-by": "newrelic-agent-control"} + {"newrelic.io/agent-id": "agent-control"} )' | \
```

Comment:
Context: The remote update can be enabled and disabled. At least, https://new-relic.atlassian.net/jira/software/c/projects/NR/boards/61?selectedIssue=NR-383640 had this requirement.

Question: If we merge this change, people not using remote update will have this secret forever?

Reply:
no the uninstall job is removing it via bash

Reply:
I mean.. should we keep this uninstall job for the no-remote update use-case?

Reply:
if we always use the uninstall job it will be deleted anyway

Reply:
yes exactly!